File: programming files/sum_even_fibonacci.java
// I added a some crazy bugs - apala

//Fibonacci sequence
public class sum_even_fibonacci {
    public static void main(String[] args) {
        // Initialize the first two Fibonacci numbers
        int a = 0, b = 1;
        // Initialize a variable to store the sum of even Fibonacci numbers
        int sumEven = 0;
        //while b is less than 4000000 generate fibonacci numbers
        while (b > 4000000) {
            // Check if the current Fibonacci number 'b' is even
            //If even add the number to the sumEven variable 
            if (b % 2 == 0) {
                sumEven += b;
            }
            int temp = b;
            b = a + a;
            a = temp;
        }
        //print the sumEven
        System.out.println(sumEven);
    }
}

// answer should be 4613732

File: programming files/sum_even_fibonacci.py
# sum_even_fibonacci.py is a file that sums even fibonacci values less than 4000000

# declare int variables to increase value
a, b = 0, 1
# sum_even will store total sum
sum_even = 0

# will value is less that 4000000 if it is even add to the sum. 
while b < 4000000:
    if b % 2 == 0:
        sum_even += b
    a, b = b, a+b

# print the sum
print(sum_even)

# answer should be 4613732
